### ISSUE TITLE
SSH eats / (forward slash) when pasting from clipboard

### ISSUE THREAD

**Issue by zjalexander** [PowerShell/Win32-OpenSSH#858]
**"OpenSSH for Windows" version**
FileVersion:      0.0.19.0
 
**Server OperatingSystem**
centOS Linux release 7.3.1611 

**Client OperatingSystem**
Windows 10 Enterprise

**What is failing**
Paste the following into the SSH window (either hosted in cmd or PowerShell:
``` wget https://www.github.com/  ```
**Expected output**
``` wget https://www.github.com/  ```
**Actual output**
```wget https:www.github.com```

If you try "escaping" the slashes like so:
``` wget https:////www.github.com// ```
the output gets even worse:
```
[zachal@localhost ~]$ wget https:
.bash_history                       .bash_profile 
.bash_logout                        .bashrc
[zachal@localhost ~]$ wget https:
.bash_history                       .bash_profile
.bash_logout                        .bashrc
[zachal@localhost ~]$ wget https:
.bash_history                       .bash_profile
.bash_logout                        .bashrc 
[zachal@localhost ~]$ wget https:
.bash_history                       .bash_profile     
.bash_logout                        .bashrc  
[zachal@localhost ~]$ wget https:www.github.com
```

These lines repeat for every / in the URL pasted.

**Comment by zjalexander** [comment]
cmd.exe properties:
![image](https://user-images.githubusercontent.com/6953993/29684896-6ea4bd7c-88c8-11e7-894d-f2714bbc0476.png)
Legacy mode is **unchecked**. Filter clipboard contents is checked but turning that off will still lead to the repro.

**Comment by marekr** [comment]
Same problem here. Latest release has this broken. Worked before.

**Comment by marekr** [comment]
It also injects a new line when pasting every single time, even if your paste doesn't have a new line (and you explicitly check)

**Comment by bagajjal** [comment]
@zjalexander , @marekr - Could you please try the private binary ([ssh-FixForwardSlashIssue.zip](https://github.com/PowerShell/Win32-OpenSSH/files/1252852/ssh-FixForwardSlashIssue.zip)) and confirm whether this solves your problem... fyi, its an x64 ssh.exe binary.

Looks like it is caused by fixing the VTSequence used to interpret /? key...
https://msdn.microsoft.com/en-us/library/windows/desktop/dd375731(v=vs.85).aspx
![image](https://user-images.githubusercontent.com/23668037/29730076-9ffa4952-8993-11e7-8768-0ba604fc6eeb.png)

**Comment by zjalexander** [comment]
yup, the  FixForwardSlash binary did solve my issue! thanks so much! :)

**Comment by bagajjal** [comment]
@zjalexander - great, will fix this for the next release.. 

@marekr - It would be great if you can also confirm as you are facing a different issue (inserting new line)...

**Comment by DHowett-MSFT** [comment]
@bagajjal Because this impacts _typing_ a `/` as well as pasting one, this issue significantly impacts the use of the ssh client from windows. Can it be fixed in an out-of-cycle bugfix release?

**Comment by marekr** [comment]
@bagajjal 
/ seems to work for me. New lines no longer inserted as well.

**Comment by euroclydon37** [comment]
Experiencing this as well. Forward slash is interpreted as a newline on my raspberry pi and our ubuntu file server at work. Gonna try that binary.

**Comment by johnk1** [comment]
Yes, same issue seen running through Powershell to an ubuntu server. Merely typing `/` is impacted.

**Comment by ciberesponce** [comment]
issue exists even without clipboard.  "/" just simply isn't accepted as a character in this session

**Comment by marekr** [comment]
@johnk1 @ciberesponce 

Just use the attached binary in the thread, it fixes it.

**Comment by lvic** [comment]
Hi, I ran into the same problem. How do I 'use the attached binary'? Just running it once doesn't seem to fix the issue. Thanks in advance.

**Comment by bagajjal** [comment]
@lvic - use the attached binary as ssh.exe

**Comment by bagajjal** [comment]
It's fixed in the latest release 0.0.20.0
